_posts/2020-02-27-talk-kubernetes-finland-metal3.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Metal³: Kubernetes Native Bare Metal Cluster Management - Maël Kimmerlin - Kubernetes and CNCF Finland Meetup"
+date: 2020-02-27
+draft: false
+categories: ["metal3", "baremetal", "talk", "conference", "kubernetes", "meetup"]
+author: Alberto Losada
+---
+
+## Conference talk: Metal³: Kubernetes Native Bare Metal Cluster Management - Maël Kimmerlin
+
+On the 20th of January at the [Kubernetes and CNCF Finland Meetup](https://www.meetup.com/Kubernetes-Finland/), Maël Kimmerlin gave a brilliant presentation about the status of the Metal³ project. 
+
+In this presentation, Maël starts giving a short introduction of the [Cluster API project](https://github.com/kubernetes-sigs/cluster-api) which provides a solid foundation to develop the Metal³ Bare Metal Operator (BMO). The talk basically focuses on the **v1alpha2** infrastructure provider features from the Cluster API. 
+
+
+> info "Information"
+> The video recording from the “Kubernetes and CNFC Finland Meetup” is composed by three talks. The video embedded starts with Maël’s talk.
+
+> warning "Warning"
+> Playback of the video has been disabled by the author. Click on play button and then on "Watch this video on Youtube" link once it appears.
+
+<iframe width="1110" height="720" style="height: 500px" src="https://www.youtube.com/embed/3k5EfIQpw-E?t=4167" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<br>
+
+During the first part of the presentation, a detailed explanation of the different Kubernetes Custom Resource Definitions (CRDs) inside Metal³ and how they are linked with the Cluster API project. As an example, the image below shows the interaction between objects and controllers from both projects:
```

Comment:
a detailed explanation of the different Kubernetes Custom Resource Definitions (CRDs) inside Metal³ is shown and also how they are linked with the Cluster API project.